app/src/main/res/values/settings_keys.xml

```diff
@@ -180,6 +181,29 @@
     <string name="app_language_key" translatable="false">app_language_key</string>
     <string name="enable_lock_screen_video_thumbnail_key" translatable="false">enable_lock_screen_video_thumbnail</string>
 
+    <string name="feed_update_threshold_key" translatable="false">feed_update_threshold_key</string>
+    <string name="feed_update_threshold_default_value" translatable="false">5</string>
+
+    <string-array name="feed_update_threshold_options">
+        <item>Always update</item>
+        <item>5 minutes</item>
+        <item>15 minutes</item>
+        <item>1 hour</item>
+        <item>6 hours</item>
+        <item>12 hours</item>
+        <item>1 day</item>
+    </string-array>
+    <string-array name="feed_update_threshold_values">
```

Comment:
I'm not sure if this is needed
```suggestion
    <string-array name="feed_update_threshold_values" translatable="false">
```

Reply:
Weblate doesn't support these so it wouldn't interfere, but it's always a good practice. Done.

Also, notice those untranslated strings above, fixed as well.